functions/package-lock.json

```diff
@@ -1174,14 +1174,6 @@
       "integrity": "sha512-3YDiu347mtVtjpyV3u5kVqQLP242c06zwDOgpeRnybmXlYYsLbtTrUBUm8i8srONt+FWobl5aibnU1030PeeuA==",
       "dev": true
     },
-    "axios": {
-      "version": "0.19.2",
-      "resolved": "https://registry.npmjs.org/axios/-/axios-0.19.2.tgz",
-      "integrity": "sha512-fjgm5MvRHLhx+osE2xoekY70AhARk3a6hkN+3Io1jc00jtquGvxYlKlsFUhmUET0V5te6CcZI7lcv2Ym61mjHA==",
-      "requires": {
-        "follow-redirects": "1.5.10"
-      }
-    },
```

Comment:
Just looked at it. It doesn't seem to be used anywhere in the project.

Reply:
*nods* I was more thinking it might be a sun-dependency of something in the functions/package-lock.json

How did this part get removed from this file? Did your do it manually? Or?

Reply:
**Note:** https://github.com/sparkletown/sparkle/pull/1139 and https://github.com/sparkletown/sparkle/pull/1140 were related to eventually fixing this regression.